README.md

```diff
@@ -23,47 +15,72 @@ We also [provide](https://github.com/moabitcoin/ig65m-pytorch/releases) converte
 |  R(2+1)D_34   | IG-65M + None       | 32x112x112 | [*r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_449a7af9.pth)                                               | [*r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ig65m_from_scratch_e304d648.pb)                                            |
 |  R(2+1)D_34   | IG-65M + Kinetics  | 32x112x112 | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_ade133f1.pth) | [*r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb*](https://github.com/moabitcoin/ig65m-pytorch/releases/download/v1.0.0/r2plus1d_34_clip32_ft_kinetics_from_ig65m_10f4c3bf.pb)  |
```

Comment:
Should just add another column with caffe model .pkl file ?

Reply:
Moved